CMakeLists.txt

```diff
@@ -239,9 +239,9 @@ link_directories(
 )
 
 if (${VSOMEIP_HOTFIX_VERSION} EQUAL 0)
-add_definitions(-DVSOMEIP_VERSION="${VSOMEIP_VERSION}")
+add_definitions(-DVSOMEIP_VERSION="${VSOMEIP_VERSION}.apexai")
 else()
-add_definitions(-DVSOMEIP_VERSION="${VSOMEIP_VERSION}.${VSOMEIP_HOTFIX_VERSION}")
+add_definitions(-DVSOMEIP_VERSION="${VSOMEIP_VERSION}.${VSOMEIP_HOTFIX_VERSION}.apexai")
 endif()
```

Comment:
nit: or, `add_definitions(-DVSOMEIP_VERSION="${VSOMEIP_VERSION}.apexai")` after the `endif()` and having the `if` to only append the hotfix version if different than 0.

Reply:
I want to avoid adding the apexai prefix in CMake-world because it is used for the versioning machinery (e.g. SOVERSION) which I think expects semver. Doing it only here affects the define only (which expects a string used only for logging)